sklearn/svm/tests/test_bounds.py

```diff
@@ -74,3 +76,40 @@ def test_ill_posed_min_c():
 def test_unsupported_loss():
     with pytest.raises(ValueError):
         l1_min_c(dense_X, Y1, loss='l1')
+
+
+# TODO test deterministic results on differing systems
+# def test_set_seed():
+#     def _test(seed, val):
+#         # TODO currently causes 99% coverage in test_bounds.py
+#         if seed is not None:
+#             set_seed_wrap(seed)
+#         x = bounded_rand_int_wrap(100)
+#         assert(x == val), 'Expected {} but got {} instead'.format(val, x)
+
+#     # TODO should be default seed for std::mt19937,
+#     # but different results on jupyter lab
+#     # _test(None, 81)
+#     _test(5489, 81)  # default seed for std::mt19937
+#     _test(0, 54)
+#     _test(4294967295, 9)  # max unsigned int size
+
+
+def test_bounded_rand_int():
+    def _test(orig_range):
+        n_pts = 1000
+        n_iter = 100
+        ks_pvals = []
+        uniform_dist = stats.uniform(loc=0, scale=orig_range)
+        # avg over multiple tests to make chance of outlier result negligible
+        for _ in range(n_iter):
+            # Deterministic random sampling
+            sample = [bounded_rand_int_wrap(orig_range) for _ in range(n_pts)]
+            res = stats.kstest(sample, uniform_dist.cdf)
+            ks_pvals.append(res.pvalue)
+        avg_pval = np.mean(ks_pvals)
+        assert(avg_pval > 0.05),\
+            'p-value of {} not > 0.05, '\
+            'therefore distribution isn\'t uniform'.format(avg_pval)
+    _test(2147483647)  # max int
+    _test(100)  # arbitrary less than max max int
```

Comment:
If I understand correctly these are two independent tests with different values for `orig_range`. You can therefore use `pytest.mark.parametrize`: 


```suggestion
_MAX_INT = 2147483647


@pytest.mark.parametrize("orig_range", [_MAX_INT, 100])
def test_bounded_rand_int(orig_range):
    n_pts = 1000
    n_iter = 100
    ks_pvals = []
    uniform_dist = stats.uniform(loc=0, scale=orig_range)
    # avg over multiple tests to make chance of outlier result negligible
    for _ in range(n_iter):
        # Deterministic random sampling
        sample = [bounded_rand_int_wrap(orig_range) for _ in range(n_pts)]
        res = stats.kstest(sample, uniform_dist.cdf)
        ks_pvals.append(res.pvalue)
    avg_pval = np.mean(ks_pvals)
    assert(avg_pval > 0.05),\
        'p-value of {} not > 0.05, '\
        'therefore distribution isn\'t uniform'.format(avg_pval)
```